jenkins/deployment/community/cortx-community-deploy.groovy

```diff
@@ -15,156 +14,168 @@ pipeline {
-                    sleep 240
-                    ssh -i cortx.pem -o StrictHostKeyChecking=no centos@"${AWS_IP}" 'pushd /home/centos/cortx-re/solutions/kubernetes && 
-                    export SOLUTION_CONFIG_TYPE='automated' && 
-                    export COMMUNITY_USE='yes' && 
-                    export CORTX_SERVER_IMAGE=$HOSTNAME:8080/seagate/cortx-rgw:2.0.0-0 && 
-                    export CORTX_DATA_IMAGE=$HOSTNAME:8080/seagate/cortx-data:2.0.0-0 && 
-                    export CORTX_CONTROL_IMAGE=$HOSTNAME:8080/seagate/cortx-control:2.0.0-0 && 
-                    sudo env SOLUTION_CONFIG_TYPE=${SOLUTION_CONFIG_TYPE} env CORTX_SERVER_IMAGE=${CORTX_SERVER_IMAGE} env CORTX_CONTROL_IMAGE=${CORTX_CONTROL_IMAGE} env CORTX_DATA_IMAGE=${CORTX_DATA_IMAGE} env COMMUNITY_USE=${COMMUNITY_USE} ./cortx-deploy.sh --cortx-cluster'
+                    export PRIMARY_PUBLIC_IP=$(cat ip_public.txt | jq '.[0]'| tr -d '",[]')
+                    export BUILD_NODE=$(cat ec2_hostname.txt | jq '.[0]'| tr -d '",[]')
+                    export SOLUTION_CONFIG_TYPE='automated'
+                    export COMMUNITY_USE='yes'
+                    export CORTX_SERVER_IMAGE="${BUILD_NODE}:8080/seagate/cortx-rgw:2.0.0-0"
+                    export CORTX_DATA_IMAGE="${BUILD_NODE}:8080/seagate/cortx-data:2.0.0-0"
+                    export CORTX_CONTROL_IMAGE="${BUILD_NODE}:8080/seagate/cortx-control:2.0.0-0"
+                    ssh -i cortx.pem -o 'StrictHostKeyChecking=no' centos@"${PRIMARY_PUBLIC_IP}" 'pushd /home/centos/cortx-re/solutions/kubernetes && export CORTX_SERVER_IMAGE='${CORTX_SERVER_IMAGE}' && export CORTX_DATA_IMAGE='${CORTX_DATA_IMAGE}' && export CORTX_CONTROL_IMAGE='${CORTX_CONTROL_IMAGE}' && sudo env SOLUTION_CONFIG_TYPE='${SOLUTION_CONFIG_TYPE}' env CORTX_SERVER_IMAGE='${CORTX_SERVER_IMAGE}' env CORTX_CONTROL_IMAGE='${CORTX_CONTROL_IMAGE}' env CORTX_DATA_IMAGE='${CORTX_DATA_IMAGE}' env COMMUNITY_USE='${COMMUNITY_USE}' ./cortx-deploy.sh --cortx-cluster'
                     popd
             '''
             }
         }
-
-        stage ('IO Sanity') {
+        stage('Basic I/O Test') {
             steps {
                 script { build_stage = env.STAGE_NAME }
                 sh label: 'IO Sanity on CORTX Cluster to validate bucket creation and object upload in deployed cluster', script: '''
                 pushd solutions/community-deploy/cloud/AWS
-                    ssh -i cortx.pem -o StrictHostKeyChecking=no centos@"${AWS_IP}" "pushd /home/centos/cortx-re/solutions/kubernetes && sudo ./cortx-deploy.sh --io-sanity"
-                popd
+                    export PRIMARY_PUBLIC_IP=$(cat ip_public.txt | jq '.[0]'| tr -d '",[]')
+                    ssh -i cortx.pem -o 'StrictHostKeyChecking=no' centos@"${PRIMARY_PUBLIC_IP}" 'pushd /home/centos/cortx-re/solutions/kubernetes && sudo ./cortx-deploy.sh --io-sanity'
+                    popd
             '''
             }
         }
     }
```

Comment:
Please add the recently added `Management Path check` also. Refer - https://github.com/Seagate/cortx-re/blob/main/jenkins/automation/kubernetes/setup-cortx-cluster.groovy#L115

Reply:
done